### PR TITLE
Make port-forward endpoint to wait for the VM

### LINKED ISSUE #62: `ssh` and `exec` commands should wait for a VM to start running
In case we call `ssh` or `exec` (#60) while a VM is still pending (not scheduled), the CLI should wait for some configurable about of time. By default, let's say 2 minutes.

### PR DESCRIPTION
**Pull request by fkorotkov**
Fixes #62

### CODE REVIEW

File: internal/controller/api_vms_portforward.go [hunk 1]
```diff
@@ -30,23 +32,40 @@ func (controller *Controller) portForwardVM(ctx *gin.Context) responder.Responde
 		return responder.Code(http.StatusBadRequest)
 	}
 
+	waitRaw := ctx.Query("wait")
+	wait, err := strconv.ParseUint(waitRaw, 10, 16)
```
Comment (edigaryev) [inline internal/controller/api_vms_portforward.go:36]: Perhaps we should check the error here? Similarly to the code a few lines above.

File: internal/controller/api_vms_portforward.go [hunk 2]
```diff
@@ -30,23 +32,40 @@ func (controller *Controller) portForwardVM(ctx *gin.Context) responder.Responde
...
 
-	if responder := controller.storeView(func(txn storepkg.Transaction) responder.Responder {
-		vm, err = txn.GetVM(name)
-		if err != nil {
-			return responder.Error(err)
-		}
+	for {
+		if lookupResponder := controller.storeView(func(txn storepkg.Transaction) responder.Responder {
+			vm, err = txn.GetVM(name)
+			if err != nil {
```
[the remark this answers is not in the document]
Comment (fkorotkov) [reply internal/controller/api_vms_portforward.go:46]: I think we don't retry here at all on any error including `ErrNotFound`. If you'll try to port forward for an non existing VM, you'll fail immediatly.

File: internal/controller/api_vms_portforward.go (hunk 1 above)
Comment (fkorotkov) [reply internal/controller/api_vms_portforward.go:36]: Good catch! I wonder why linter didn't catch it. Fixed in [69ab796](https://github.com/cirruslabs/orchard/pull/65/commits/69ab796870b3e4664e4ece3db0b0397770726aa8)

File: pkg/client/vms.go [hunk 3]
```diff
@@ -101,9 +101,17 @@ func (service *VMsService) Delete(ctx context.Context, name string) error {
 	return nil
 }
 
-func (service *VMsService) PortForward(ctx context.Context, name string, port uint16) (*websocket.Conn, error) {
+func (service *VMsService) PortForward(
+	ctx context.Context,
+	name string,
+	port uint16,
+	waitSeconds uint16,
```
Comment (edigaryev) [inline pkg/client/vms.go:108]: Should the wait time here be optional? Because it is in the API.

Comment (fkorotkov) [reply pkg/client/vms.go:108]: IMO waiting by default make sense for users who will call `ssh` immediately after VM creation:

```bash
orchard create vm --image ghcr.io/cirruslabs/macos-ventura-base:latest --headless=false ventura-base
orchard ssh vm ventura-base
```

I think it's better UX.

**Review by edigaryev — APPROVED** [review]

**Pull request merged**